modules/indexing/src/test/java/org/apache/ignite/internal/processors/query/stat/StatisticsConfigurationTest.java

```diff
@@ -231,7 +231,9 @@ public void checkClientNode() throws Exception {
 
         IgniteEx client = startClientGrid("cli");
 
-        collectStatistics(SMALL_TARGET);
+        Thread.sleep(1000);
```

Comment:
is this `Thread.sleep` necessary?

Reply:
Replaced with awaitPartitionMapExchange with slightly bigger collection timeout to make tests more stable.